app/views/maps/index.html.erb

```diff
@@ -1 +1 @@
-<div class="spinner"></div>
+<div class="spinner spinner-big"></div>
```

Comment:
💥

Reply:
damn, thx for this! it's supposed to be `spinner--big` :D